### DISCUSSION TITLE
How to set the (git)commit time to be the time of last `jj desc` instead of the initialization time(i.e. `jj new`)

### DISCUSSION THREAD

**Discussion by Ziqi-Yang** [jj-vcs/jj#3112] [Q&A] [answered]
I always have a new empty slot for new work. And I do this style to push git repo
```bash
# there is already a empty slot, the time shown in the git log is the time that this slot created
# edit file
jj desc
jj new
jj branch set main -r @-
jj git push
```
And the time displaying in git log is the time that the new slot created. I want to have time displayed as last `jj desc`.

**Comment by martinvonz** [comment, marked as answer]
See `git help log` for how to tell it to show the committer timestamp instead of the author timestamp.

I think you might also be asking if we could update the author date later. See #2000 for that.
